setup.py

```diff
@@ -41,31 +41,23 @@ def run(self):
     namespace_packages=['chevah'],
     packages=find_packages('.'),
     package_data={
-        'chevah.github_hooks_server': ['static/*', 'author-aliases.txt'],
+        'chevah.github_hooks_server': ['author-aliases.txt'],
         },
-    scripts=['scripts/start-chevah-github-hooks.py'],
     install_requires=[
-        'klein==17.2',
         'toml',
-        # We keep an older version of python is use.
-        'Twisted==15.5.0.chevah1',
-        'github3-py==1.0.0.gitc82e90e',
-        # We don't depended directly on them but github3.py
-        # dependencies are bad.
-        'urllib3',
-        'chardet',
-        'certifi',
-        'idna',
+        'github3.py',
+        'azure-functions',
+        'azure-functions-worker',
         ],
-    extras_require = {
+    extras_require={
         'dev': [
             'mock',
             'nose',
             'pyflakes',
             'pep8',
```

Comment:
here we might want to add `nodeenv` and have a script that will call nodenv with a specific node version

Reply:
> and have a script that will call nodenv with a specific node version

I added `nodeenv` here, and added the "Development" section in the readme, but do you also want a script creating the environment?

Reply:
no. I think that you create the env only once. It's ok to have it only in the readme .

Reply:
I will look at `nodeenv` options, to see if I can move everything to one `node_modules` folder.

Reply:
I could get `serverless-azure-functions` to install globally in the venv, but it won't run from there.

```
Error ---------------------------------------------------
 
  Error: spawn /home/dan/Documents/Work/ProAtria/github-hooks-server/node_modules/.bin/func ENOENT
      at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
      at onErrorNT (node:internal/child_process:475:16)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
```